docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -621,6 +711,134 @@ This command will activate the ``regional_workflow`` conda environment. The user
    source ~/.bashrc
    conda activate regional_workflow
 
+.. _MacConfig:
+
+Configuring an Experiment on MacOS
+------------------------------------------------------------
+
+In principle, the configuration process for MacOS systems is the same as for other systems. However, the details of the configuration process on MacOS require a few extra steps. 
+
+.. _MacMorePackages:
+
+Install Additional Packages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Check the version of bash, and upgrade it if it is lower than 4. Additionally, install the ``coreutils`` and ``gsed`` packages:
+
+.. code-block:: console
+
+   bash --version
+   brew upgrade bash
+   brew install coreutils
+   brew install gsed
+   
+The ``gsed``, or GNU streamline editor, can be set as a default for ``sed``. Add a ``gnubin`` directory to your $PATH variable in ``.bashrc`` : 
```

Comment:
If going back to re-build hpc-stack, leaving the change to gsed causes a failure in the openmpi build, so not sure if its a good idea to suggest adding the PATH change to .bashrc.

Reply:
@willmayfield - the openmpi/4.1.2 is suggested to be built on M1 (arm64) platforms, while the mpich/3.3.2 is for x86_64.

Reply:
@natalie-perlin Sorry, yeah, I mean for any time you have hpc-stack build mpi.

Reply:
Let me test that. May I ask what type of Mac are you using (x86_64 or M1) and bash version?.. 
If you have a chance to also share on what type of issue the 'gnu-sed/ is causing for the  MPI libraries to build, would be awesome, too! 
What if 'sed' binary is left untouched (no new link to 'gsed' created), and 'gsed' command is used in other documentation chapters for MacOS as needed?
The problem with 'sed' was apparent with bash v.3.x.x that came with my Mac M1. Updating 'sed' to 'gnu-sed' and updating 'bash' solved the issue. Or is it the ./gnubin/ library that may have other conflicting binaries?

Technically, I installed the HPC stack first, and _then_ updated 'sed' to 'gsed' to make bulk changes to the  ./regional_workflow/ush/wrappers/*.sh scripts for the SRW app. It was surely needed on the M1 Mac. So I would need to test the following in order to understand your comment on the issue:
1) Rebuild the MPI libraries after 'gsed' updates (and bash update) were implemented on  i) M1 and ii) x86_64
 2) Test building different MPI libraries, before or after the gnu-sed updates. 
 3) Removing the /gnubin/ from being a prefix to $PATH, and maybe make it a suffix (further down the search line)

Please let me know any comments!

Reply:
Sure. I'm testing on an M1 with bash 5.1.16. The problem I had in hpc-stack was line 76 didn't execute in libs/build_mpi.sh. And yeah, I'm not sure the best solution, just that it could cause some confusion if someone runs the app then re-builds hpc-stack later.

Reply:
Super!! This is very helpful info, thank you!!. I'll keep that in mind.

Reply:
@willmayfield @gspetro-NOAA 
Solution found, an easy fix. No need to install gnu-sed. The `sed` coming with MacOS works perfectly if option "-e" is added, e.g.,

`$ sed -i -e 's/QUAD_PRECISION\"  ON)/QUAD_PRECISION\" OFF)/' CMakeLists.txt`
or
`$ sed -i -e 's/bin\/sh/bin\/bash/g' *sh`

Directions to install `gnu-sed` could be removed from the documentation.

Reply:
@gspetro-NOAA 
Another thing that needs to be added is openssl@3 installation via brew. I rearranged the sub-sections in "Prerequisites for the HPC-stack" to include it. (1.3 and 1.4 in the 
https://docs.google.com/document/d/1J2i0i9RrY65sKE2nvYQ4Vy38mw2Yda1NkOZARpzSIHw )

Reply:
@natalieperlin To confirm, the openssl@3 should be added to HPC-Stack docs, not this PR, right? Also, in the first of the two `sed` command options, there's no g after OFF)/ Is that correct? Or did you leave it off by accident?

Reply:
@gspetro-NOAA - yes, a need to install openssl@3 could be added to the HPC-stack docs for the Mac build, may not need to be this PR. 
And yes, I removed the letter 'g', which is for 'global' changes throughout the file, because this string is found only in a single line in the CMakeLists.txt ! So having 'g' is redundant.